cmd/protecodeExecuteScan.go

```diff
@@ -429,21 +434,41 @@ func correctDockerConfigEnvVar(config *protecodeExecuteScanOptions) {
 }
 
 func getTarName(config *protecodeExecuteScanOptions) string {
+
 	// remove original version
 	fileName := strings.TrimSuffix(config.ScanImage, ":"+config.Version)
 	// remove sha digest if exists
 	sha256 := "@sha256"
 	if index := strings.Index(fileName, sha256); index > -1 {
 		fileName = fileName[:index]
 	}
-	// append trimmed version
-	if version := handleArtifactVersion(config.Version); len(version) > 0 {
-		fileName = fileName + "_" + version
+
+	version := getProcessedVersion(config)
+	if len(version) > 0 {
+		fileName = fileName + " - " + version
```

Comment:
I feel we should avoid using blanks but besides anyway stick to the former format, otherwise we risk breaking audit history via file name, right?